topics/google/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: Google
+github_url: https://github.com/google
+logo: gradle.png
```

Comment:
Oops, can you make this `google.png`? Note for any future contributions: this was caught by the CI tests which can give you quicker feedback than I can!